css/at-rules/property.json

```
@@ -48,6 +48,153 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "inherits": {
+          "__compat": {
+            "description": "inherits",
```

suggestion: if we're going to have description, we ought to expand on the the key for this feature

```suggestion
            "description": "<code>inherits</code> descriptor",
```